pkgs/games/openra/builds.nix

```diff
@@ -0,0 +1,34 @@
+pkgs:
+
+let
+  f = import ./common.nix;
+  common = f (builtins.intersectAttrs (builtins.functionArgs f) pkgs // {
+    inherit (pkgs) stdenv;
+    lua = pkgs.lua5_1;
+    inherit (pkgs.gnome3) zenity;
+  });
+
+  buildOpenRASet = f: args: builtins.mapAttrs (name: value: if builtins.isFunction value then value name else value) (f ({
+    inherit (pkgs) fetchFromGitHub;
+    extraPostFetch = ''
+      sed -i 's/curl/curl --insecure/g' $out/thirdparty/{fetch-thirdparty-deps,noget}.sh
+      $out/thirdparty/fetch-thirdparty-deps.sh
+    '';
+  } // args));
+
+in rec {
+  buildOpenRAEngine = { name ? null, version, description, homepage, mods, src, installExperimental ? "" }@engine:
+    let builder = name: pkgs.callPackage ./engine.nix (common // {
+      engine = engine // { inherit name installExperimental; };
```

Comment:
Seems like here you override `engine.installExperimental` with `engine.installExperimental`

Reply:
Nope, that is not how destructering of attrs work in Nix:
```nix
(({ foo ? null }@attrs: attrs) {}).foo
```
`error: attribute 'foo' missing, at (string):1:1`
```nix
(({ foo ? null }@attrs: attrs // { inherit foo; }) {}).foo
```
`null`